### DISCUSSION TITLE
MPL and simple-udp

### DISCUSSION THREAD

**Discussion by rahav** [contiki-ng/contiki-ng#1710] [Q&A] [answered]
@edrose and @g-oikonomou Do you foresee any problem using MPL with simple-udp instead of Contiki UDP?

**Comment by edrose** [comment, marked as answer]
I can see no reason why not. MPL doesn't care what is in the layer above, as long as there is enough space in the IP buffer to fit the MPL header in.

**Reply by rahav** [reply]
You were correct. Thank you.

To test I migrated `examples/multicast` and ran regression tests. There's perfect match. Let me know if you think that the community will benefit from having this example alongside the existing, and I'll be happy to start a pull request.
